engine/src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -197,25 +197,39 @@ trait CromwellApiService {
       case b: BodyPart => b.toStrict(duration).map(strict => b.name -> strict.entity.data)
     }.runFold(Map.empty[String, ByteString])((map, tuple) => map + tuple)
 
+    def toResponse(workflowId: WorkflowId): WorkflowSubmitResponse = {
+      WorkflowSubmitResponse(workflowId.toString, WorkflowSubmitted.toString)
+    }
+
+    def askSubmit(command: WorkflowStoreActor.WorkflowStoreActorSubmitCommand): Route = {
+      // NOTE: Asks were already here. Guessing this was a KISS port of per-request via:
```

Comment:
> that's how one does akka http

Can you provide a doc to link to that states this and I'll include it? The link I provided in the comment has an example that _doesn't_ use `ask`, after of course mentioning `ask` a "first way".

I'm not actually convinced that the Workflow Submission needs to be an Actor anymore, and the `ask` seems to be a retrofit to talk to an akka Actor wrapper around calls to scala Futures.

FYI- Clio hasn't matured to prod status, yet, but wasn't ever using the spray-esque per-request model. It only used scala Futures and (in-one-case) an akka-stream. Everything on first glance seemed much cleaner imho, but everything looks that way when there's not much code there. ;)